docs/source/diagnostics-performance.rst

```diff
@@ -0,0 +1,16 @@
+Diagnostic and Performance
+==========================
+
+This section contains resources to help you debug, obtain diagnostic and understand
```

Comment:
again, I think that "diagnostics" is slightly more idiomatic english here.  A diagnostic is a particular test that you do.  Diagnostics are the results of such a test (or at least that's my read of the word)